app/src/main/java/com/imdbmovieapp/data/remote/dto/popularMoviesDto/PopularMoviesDto.kt

```diff
@@ -0,0 +1,8 @@
+package com.imdbmovieapp.data.remote.dto.popularMoviesDto
+
+data class PopularMoviesDto(
+    val page: Int,
+    val moviesResults: List<MoviesResult>,
+    val total_pages: Int,
+    val total_results: Int
```

Comment:
Try to sort properties by their importance (for example, in my opinion, `movieResults` property is more important than `page`) and move it above or below others